### ISSUE TITLE
Problems with installing Git for Windows

### ISSUE THREAD

**Issue by Shakil-Shahadat** [git-for-windows/git#523]
1. Opens in background, tried couple of times with same result.

2. While trying to install first time it said it can't create folder 'C:\Program Files\Git\tmp', though it already asked for administrative privilege and I gave it. In later attempts, it didn't even ask for administrative privilege.

**Comment by TaleTN** [comment]
Same here (on Win7 Pro x64). When I ran the installer as administrator it seems to install just fine, although it didn't seem to remember my installer options (but I guess that is because I ran it as administrator, right?).

**Comment by mikefourie** [comment]
Having the same issues. Now installing to the new default which appears to be the users hive, though this appears to be hung after 20 minutes+

**Comment by dscho** [comment]
Could you try to uninstall Git completely, then *manually remove* `C:\Program Files\Git` and then reinstall (not as admin)?

**Comment by TaleTN** [comment]
Sure. After uninstalling I reinstalled into C:\Users\...\AppData\Local\Programs\Git, but then the installer appeared to be stuck. But after ending the task, uninstalling again, rebooting, and then running the installer again I have managed to successfully install Git into C:\Users\...\AppData\Local\Programs\Git. So I guess this seems to works as intended, as long as you haven't installed into C:\Program Files\Git before. Good I guess, thanks.

However, IMHO you shouldn't install binaries into AppData, so I think I would prefer to install into C:\Program Files\Git. I guess this means that I have to run the installer as administrator from now on, but so be it.

**Comment by mikefourie** [comment]
same as what TaleTN said...

**Comment by dscho** [comment]
> you shouldn't install binaries into AppData

@TaleTN *I* don't.

**Comment by TaleTN** [comment]
@dscho Don't what?

**Comment by dscho** [comment]
@TaleTN read again what you wrote: "you should'nt install binaries into AppData" and then read again what I wrote: "I don't".

**Comment by TaleTN** [comment]
Ah, I see... Well, the current installer does, and by default, so I would suggest changing that to a more sensible default then (I guess like it was before). Thanks!

**Comment by dscho** [comment]
> the current installer does

No, it does not. At least if you run it as an administrator (and if I am not mistaken, also under certain other circumstances). It installs into `C:\Program Files\Git` by default

It only falls back to AppData if the current user *cannot write to `C:\Program Files`*.

**Comment by jzp74** [comment]
I'm having the same problem: 2.6.2 installed by default correctly in C:\Program Files\Git
Removed 2.6.2 and installed 2.6.3 (with same user, who has admin rights) the installer cannot (and will not) install in C:\Program Files\Git

**Comment by TaleTN** [comment]
@dscho Yes, you are right. However, do note that by default a normal Windows user can't write to C:\Program Files, so to do a "normal" installation most users will have to run the installer as administrator now. This isn't a huge issue, but IMHO it is a nuisance, and apparently (and understandably, again IMHO) it leads to questions.

**Comment by dscho** [comment]
@TaleTN yes, I understand that it is a nuisance and that it is nice to vent. Please note, though, that the conversation you force on me does only one thing: delay the fix. (Hint: you could have helped working on a resolution; or you could have let me work on the resolution instead of spending time in this here bug tracker.)

**Comment by TaleTN** [comment]
@dscho Right!
https://github.com/git-for-windows/build-extra/pull/88
